packages/database/src/utils/__tests__/encryption.test.ts

```diff
@@ -0,0 +1,123 @@
+import { testData } from '../__fixtures__';
+import { createHash, decryptData, encryptData } from '../encryption';
+
+const { hashing, encryption, decryption } = testData;
+
+describe('createHash', () => {
+  test('Should create hash with valid values', () => {
+    const { valid: validTestCases } = hashing;
+
+    validTestCases.forEach(item => {
+      if (item.expectedHashKey && item.message) {
+        expect(createHash(item.message).toString('hex')).toEqual(
+          item.expectedHashKey,
+        );
+      }
+    });
+  });
```

Comment:
This test case (and other such tests) should be refactored as mentioned here: https://github.com/Cypherock/cypherock-cysync/pull/454#discussion_r1586244463